modules/syslogformat/syslog-format.c

```diff
@@ -1170,7 +1171,7 @@ log_msg_parse_syslog_proto(const MsgFormatOptions *parse_options, const guchar *
     {
       return log_msg_parse_legacy(parse_options, data, length, self, position);
     }
-
+  msg_debug("Setting macros on IETF syslog message");
   if (!log_msg_parse_skip_space(self, &src, &left))
```

Comment:
similar to what I had on BSD

Reply:
Thinking it again there is no additional value in this message, I will delete it.